project/plugins.sbt

```diff
@@ -1,3 +1,5 @@
+resolvers += "typesafe" at "http://repo.typesafe.com/typesafe/releases"
+
```

Comment:
This doesn't do anything, so you can remove it.

Reply:
@ashawley Yes, thank you very much for the fix suggestion about the extra SBT_OPTS, I was puzzling what's went wrong and you saved my day!

And yes the typesafe repo thing above didn't do anything, I'll remove it and after it builds green again I'll remove the [DO-NOT-MERGE] tag for this PR.

Thank you again for the fix!